src/MongoDB/ReadPreference.c

```diff
@@ -369,6 +375,31 @@ static PHP_METHOD(ReadPreference, getTagSets)
 	}
 } /* }}} */
 
+static const char* php_phongo_readpreference_get_mode_string(mongoc_read_mode_t mode TSRMLS_DC) /* {{{ */
+{
+	switch (mode) {
+		case MONGOC_READ_PRIMARY:
+			return "primary";
+			break;
```

Comment:
You can omit the `break` after a `return` within a `case` block. It's technically an unreachable statement anyway.